repository/src/test/java/org/alfresco/repo/security/authentication/AuthenticationTest.java

```diff
@@ -2076,44 +2074,49 @@ public void testRehashedPasswordOnAuthentication() throws Exception
 
     /**
      * For on premise the default is MD4, for cloud BCRYPT10
-     *
-     * @throws Exception
      */
-    public void testDefaultEncodingIsMD4() throws Exception
+    public void testDefaultEncodingIsMD4()

```

Comment:
As "MD4" is not the default encoding anymore, maybe method name should be fixed?